backend/typescript/models/donor.model.ts

```diff
@@ -0,0 +1,25 @@
+import { Column, DataType, Model, Table, ForeignKey, BelongsTo } from "sequelize-typescript";
+import { DonorRole } from "../types"
+import User from "./user.model"
+
+@Table({ tableName: "donors" })
+export default class Donor extends Model {
+  @ForeignKey(() => User)
+  @Column({ type: DataType.INTEGER })
+  user_id!: number;
+
+  @BelongsTo(() => User)
+  user!: User;
+
+  @Column({ type: DataType.ENUM("LocalBusiness", "IndividualDonor") })
```

Comment:
```suggestion
  @Column({ type: DataType.ENUM("Local Business", "Individual Donor") })
```

Reply:
Enums should generally not have spaces just like variable naming convention, so it would be better if we keep it without spaces